relay/sources/Makefile

```diff
@@ -135,6 +135,7 @@ install: build
 	install -m 644 rudder-relay-apache $(DESTDIR)/etc/sysconfig/rudder-relay-apache
 	install -m 644 rudder-relay.cron $(DESTDIR)/etc/cron.d/rudder-relay
 	install -m 644 rudder-relay.sudo $(DESTDIR)/etc/sudoers.d/rudder-relay
+	install -m 755 relay-cleanup.sh $(DESTDIR)/opt/rudder/bin/relay-cleanup
```

Comment:
why not remove the .sh in the source too ?

Reply:
done